_data/simple-icons.json

```diff
@@ -2742,6 +2742,11 @@
                 "type": "CC-BY-3.0"
             }
         },
+        {
+            "title": "Expensify",
+            "hex": "0185FF",
+            "source": "https://use.expensify.com/press-kit"
```

Comment:
Thanks, @dhruvkb. If you can duplicate that for the `guidelines` entry then we'll be good to go
```suggestion
            "source": "https://use.expensify.com/press-kit",
            "guidelines": "https://use.expensify.com/press-kit"
```

Reply:
Done!